Source/Core/Plugins/EventProcessor/Default/20_ErrorPlugin.cs

```diff
@@ -39,6 +39,9 @@ public class ErrorPlugin : EventProcessorPluginBase {
 
             error.Data[Error.KnownDataKeys.TargetInfo] = targetInfo;
 
+            if (context.StackSignatureData.Count > 0)
```

Comment:
Any specific reason for this change? I'm a bit worried about it as it could change the behavior of how things were working previously? Should we be checking to see if stack signature data contains our known key here instead?

Is there any chance we could add some unit tests for this? I could even add them..